mvp-0/values-observer.yaml

```diff
@@ -12,14 +26,22 @@ thanos:
       type: NodePort
       nodePorts:
         remote: 30291
+  compactor:
+    enabled: true
+    retentionResolutionRaw: 2d
+    retentionResolution5m: 10d
+    retentionResolution1h: 15d
+    extraFlags:
```

Comment:
What about `delete-delay` parameter as mentioned in [thanos tuning docs](https://github.com/SovereignCloudStack/k8s-observability/blob/main/docs/thanos-tuning.md)? Or we have some default there ?

Reply:
There is default as it is mentioned in that [thanos tuning doc](https://github.com/SovereignCloudStack/k8s-observability/blob/main/docs/thanos-tuning.md)

Reply:
Thank you